src/codewhisperer/activation.ts

```diff
@@ -94,7 +93,6 @@ export async function activate(context: ExtContext): Promise<void> {
 
     context.extensionContext.subscriptions.push(
         signoutCodeWhisperer.register(auth),
-        showManageCwConnections.register(),
```

Comment:
This command called the main `showManageConnections()` underneath but with some pre-set CW args. The issue was that this causes 2 different commands for telemetry instead of having 1. So we will use the original `showManageConnections()` instead.